src/JPEGView/JPEGView.vcxproj

```diff
@@ -112,7 +112,7 @@
       <DebugInformationFormat>EditAndContinue</DebugInformationFormat>
       <RuntimeTypeInfo>true</RuntimeTypeInfo>
       <OpenMPSupport>false</OpenMPSupport>
-      <LanguageStandard>stdcpp17</LanguageStandard>
+      <LanguageStandard>stdcpp20</LanguageStandard>

```

Comment:
What new language feature is used in the new code that requires cpp20?

Reply:
In fact ISO C++20 Standard is not required for my changes. I can revert it back or left it as is to have latest language version support in the project.

Reply:
Reverted it back,